chapters.md

```diff
@@ -3,31 +3,82 @@
 
 # Chapters
 
-## Edition and Chapter Flags
+The Matroska Chapters system can consist of`Simple Chapters` where a chapter start time is used as marker in the timeline only. The system can be more complex with `Ordered Chapters` where a chapter end time stamp is additionally used or much more complex with `Linked Chapters`. The Matroska Chapters system can also have a menu structure, borrowed from the DVD menu system, or have it's own Native Matroska menu structure.
 
-### Chapter Flags
+## EditionEntry
 
-Two `Chapter Flags` are defined to describe the behavior of the `ChapterAtom Element`: `ChapterFlagHidden` and `ChapterFlagEnabled`.
+The `EditionEntry` is also called an `Edition`.  
+An `Edition` is a set of `ChapterAtoms` and there MUST be at least one `ChapterAtom`.  
```

Comment:
An `Edition` (aka `EditionEntry`) is more than a set of `Chapter Atoms` since it also include Edition Flags. I suggest "An \`Edition\` contains a set of \`Edition Flags\` and MUST contain at least one \`ChapterAtom Element\`".